presto-docs/src/main/sphinx/connector/hive.rst

```diff
@@ -447,6 +447,13 @@ The following operations are not supported when ``avro_schema_url`` is set:
 * Using partitioning(``partitioned_by``) or bucketing(``bucketed_by``) columns are not supported in ``CREATE TABLE``.
 * ``ALTER TABLE`` commands modifying columns are not supported.
 
+Procedures
+----------
+
+* ``system.create_empty_partition(schema, table, partition_cols, partition_vals)``
```

Comment:
Since procedures allow specifying arguments by name, we need to document the exact names here.